### DISCUSSION TITLE
Vuetify sortable list or data-table

### DISCUSSION THREAD

**Discussion by ghost** [vuetifyjs/vuetify#14120] [Ideas]
I think it would be a good idea to have a functionality to move the order of certain elements in tables or lists. 

If you want to reorder a specific image collection for example. easily drag your element to the desired location in the list and the order of the array would be changed. I can think of many use cases where this could be convenient.

Here is an example from the Jquery UI Library https://jqueryui.com/sortable/

**Comment by ghost** [comment]
@johnleider any ideas on this?

**Comment by KaelWD** [comment]
https://github.com/SortableJS/Vue.Draggable

https://github.com/rlemaigre/Easy-DnD
